runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangeConfig.java

```diff
@@ -47,14 +47,25 @@ default boolean quickLookup()
 	@ConfigItem(
 		position = 2,
 		keyName = "enableNotifications",
-		name = "Enable Notifications",
+		name = "Notify on offer update",
 		description = "Configures whether to enable notifications when an offer updates"
 	)
 	default boolean enableNotifications()
 	{
 		return true;
 	}
 
+	@ConfigItem(
+		position = 2,
```

Comment:
Just to double check on line 48 the config is also position = 2, is this an issue?
![image](https://github.com/runelite/runelite/assets/59254672/9f54fd28-9b95-4607-9bc4-45346567767f)

Reply:
It orders them alphabetically then which is fine.